test/appium/tests/test_wallet.py

```diff
@@ -120,3 +120,18 @@ def test_eth_and_currency_balance(self):
             errors.append('Balance %s is not equal to the expected %s' % (wallet_balance, balance))
         wallet.verify_currency_balance(eth_rate, errors)
         assert not errors, 'errors occurred:\n{}'.format('\n'.join(errors))
+
+    @pytest.mark.wallet
```

Comment:
@antdanchenko any opinion on this appium test ?

Reply:
Ping @antdanchenko

Reply:
@antdanchenko Could you get more feedback on that point? Or can you only get some after it's merged?

Reply:
I think we should not have such tests at the end-end level. Instead, a unit test triggering a failure and checking what method is called (`show-popup "Error"`) or something like that will probably be enough. 

As @antdanchenko said, it probably will not work in the cloud (saucelabs).

Reply:
I've also run it locally and it didn't work correctly. No error is shown because device checks tokens prices after going offline (`self.driver.set_network_connection(1)`) which means the request is not being sent at all.

Reply:
@foopang let me rework this test for you. I will extend it a bit and make sure it works in our cloud. In the meantime you can focus on the issues I mentioned in my previous comment. Let me know how it goes. Thanks!